chat-plugins/wifi.js

```diff
@@ -289,8 +291,16 @@ var commands = {
 		for (var i in giveaway.answers) {
 			answers.push(giveaway.answers[i]);
 		}
+		var format = [];
```

Comment:
this whole block could be written in a single line:
`var format = ((answers.length === 1) ? 'answer is' : 'answers are').split(' ');`